fastlane/Fastfile

```diff
@@ -66,6 +67,21 @@ platform :android do
     gradle(task: ":test-apps:testpurchasesuiandroidcompatibility:emergeUploadSnapshotBundleDebug")
   end
 
+  desc "Emerge size tests"
+  lane :emerge_sdk_size_tests do
+     gradle_properties = {
+       "signing.keyId" => ENV['GPG_SIGNING_KEY_ID_NEW'],
+       "signing.password" => ENV['GPG_SIGNING_KEY_PW_NEW'],
+       "signing.secretKeyRingFile" => "/home/circleci/.gnupg/secring.gpg",
+       "RELEASE_SIGNING_ENABLED" => true
+     }
+     gradle(task: ":purchases:publishToMavenLocal", properties: gradle_properties)
+     gradle(task: ":ui:revenuecatui:publishToMavenLocal", properties: gradle_properties)
+     Dir.chdir("../test-apps/sdksizetesting/app") do
```

Comment:
This is a bit weird... I need to enter the module folder, because fastlane's `gradle` command searches the `gradlew` executable in the parent folder